File: Musixmatch_lyrics_API_App/app/src/main/java/alexejantonov/com/musixmatch_lyrics_api_app/db/DataBaseContract.java
package alexejantonov.com.musixmatch_lyrics_api_app.db;

class DataBaseContract {

	class ArtistsTable {

		static final String ARTISTS_TABLE_NAME = "artists";

		static final String COLUMN_ARTIST_ID = "artistId";
		static final String COLUMN_ARTIST_NAME = "name";
		static final String COLUMN_ARTIST_TWITTER = "twitter";
		static final String COLUMN_ARTIST_TOP_CHART_COUNTRIES = "countries";
	}

	class TracksTable {
		static final String TRACKS_TABLE_NAME = "tracks";

		static final String COLUMN_TRACK_ID = "trackId";
		static final String COLUMN_TRACK_NAME = "name";
		static final String COLUMN_TRACK_ALBUM = "album";
		static final String COLUMN_TRACK_ARTIST_ID = "artistId";
		static final String COLUMN_TRACK_ALBUM_COVER = "albumCover";
	}
}

File: Musixmatch_lyrics_API_App/app/src/main/java/alexejantonov/com/musixmatch_lyrics_api_app/db/DataBaseContract.kt
package alexejantonov.com.musixmatch_lyrics_api_app.db

class DataBaseContract {

  object ArtistsTable {
    const val ARTISTS_TABLE_NAME = "artists"

    const val COLUMN_ARTIST_ID = "artistId"
    const val COLUMN_ARTIST_NAME = "name"
    const val COLUMN_ARTIST_TWITTER = "twitter"
    const val COLUMN_ARTIST_TOP_CHART_COUNTRIES = "countries"
  }

  object TracksTable {
    const val TRACKS_TABLE_NAME = "tracks"

    const val COLUMN_TRACK_ID = "trackId"
    const val COLUMN_TRACK_NAME = "name"
    const val COLUMN_TRACK_ALBUM = "album"
    const val COLUMN_TRACK_ARTIST_ID = "artistId"
    const val COLUMN_TRACK_ALBUM_COVER = "albumCover"
  }
}